cpp/include/cuspatial/hausdorff.hpp

```diff
@@ -0,0 +1,23 @@
+#pragma once
+#include <cudf/cudf.h>
+
+namespace cuSpatial {
+
+/**
+ * @Brief compute Hausdorff distances among all pairs of a set of trajectories
+ * https://en.wikipedia.org/wiki/Hausdorff_distance
+
+ * @param[in] coord_x: x coordinates of the input trajectroies
+ * @param[in] coord_y: y coordinates of the input trajectroies
+ * @param[in] traj_cnt: numbers of vertices of the set of trajectories;
+ * also used to compute the starting offsets of the trjajectories in coord_x/coord_y arrays
+
+ * @returns a flatted (1D) vector of all-pair direted Hausdorff distances among trajectories (i,j)
```

Comment:
```suggestion
 * @returns a flattened (1D) column of all-pairs directed Hausdorff distances among trajectories (i,j)
```

How are the outputs ordered in terms of (i, j)? Document this.